components/programmes/ConditionsOfJoining.tsx

```diff
@@ -23,58 +25,75 @@ export function ConditionsOfJoining({
   programmeName
 }: ConditionsOfJoiningProps) {
   return conditionsOfJoining.signedAt ? (
-    <dl className="nhsuk-summary-list" data-cy="signedCoj">
-      <div className="nhsuk-summary-list__row">
-        <dt className="nhsuk-summary-list__key">Signed</dt>
-        <dd className="nhsuk-summary-list__value" data-cy="cojSignedDate">
+    <SummaryList data-cy="signedCoj">
+      <SummaryList.Row>
+        <SummaryList.Key>Signed</SummaryList.Key>
+        <SummaryList.Value data-cy="cojSignedDate">
           {DateUtilities.ToLocalDate(conditionsOfJoining.signedAt)}
-        </dd>
-      </div>
-      <div className="nhsuk-summary-list__row">
-        <dt className="nhsuk-summary-list__key" style={{ borderBottom: 0 }}>
-          Version
-        </dt>
-        <dd
-          className="nhsuk-summary-list__value"
+        </SummaryList.Value>
+      </SummaryList.Row>
+      <SummaryList.Row>
+        <SummaryList.Key style={{ borderBottom: 0 }}>Version</SummaryList.Key>
+        <SummaryList.Value
           style={{ borderBottom: 0 }}
           data-cy="cojSignedVersion"
         >
           {CojUtilities.getVersionText(conditionsOfJoining.version)}
-        </dd>
-      </div>
-    </dl>
+        </SummaryList.Value>
+      </SummaryList.Row>
+      <SummaryList.Row>
+        <SummaryList.Actions style={{ borderBottom: 0 }}>
+          <Link
+            to={`/programmes/${programmeMembershipId}/sign-coj`}
+            onClick={() =>
+              setCojState(
+                programmeMembershipId,
+                programmeName,
+                conditionsOfJoining.signedAt
+              )
+            }
+            data-cy={`cojViewBtn-${programmeMembershipId}`}
+          >
+            View
+          </Link>
+        </SummaryList.Actions>
+      </SummaryList.Row>
+    </SummaryList>
   ) : (
-    <dl className="nhsuk-summary-list" data-cy="unsignedCoj">
-      <div className="nhsuk-summary-list__row">
-        <dd
-          className="nhsuk-summary-list__value"
-          style={{ borderBottom: 0 }}
-          data-cy="cojStatusText"
-        >
+    <SummaryList data-cy="unsignedCoj" noBorder>

```

Comment:
@EdwardBarclay couple of issues with this fix
 1. It removes all the borders, which I don't think we want.
 2. We need to make sure we fix the issue for both the sign and view scenarios